basculehelper/basculeHelper.go

```diff
@@ -0,0 +1,510 @@
+package basculehelper
+
+//Deprecated: this is a bascule helper package that uses older bascule functions from webpa-common in order to implement
+//sallust and zap logger in scytale
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"emperror.dev/emperror"
```

Comment:
We should probably coalesce around the go.uber.org/multierr package.  emperror is fine, but it would be nice to just use one error library everywhere.